packages/playground/website/playwright/e2e/blueprints.spec.ts

```diff
@@ -149,3 +149,13 @@ test('PHP Shutdown should work', async ({ website, wordpress }) => {
 	await website.goto(`/#${JSON.stringify(blueprint)}`);
 	await expect(wordpress.locator('body')).toContainText('Dashboard');
 });
+
+test('should log the user in', async ({ website, wordpress }) => {
```

Comment:
There's an implicit assumption here that the default Blueprint, without the `login` parameter, logs the user in. Let's make these explicit when possible – either in the test name or by declaring that parameter in the Blueprint itself.